src/pages/Beta/Pool/EarnWidget/AddLiquidity/index.tsx

```diff
@@ -264,15 +264,15 @@ const AddLiquidity = ({ currencyA, currencyB, onComplete, onAddToFarm, type }: A
                     width={approvalA !== ApprovalState.APPROVED ? '48%' : '100%'}
                     loading={approvalB === ApprovalState.PENDING}
                     loadingText={`${t('swapPage.approving')} ${currencies[Field.CURRENCY_B]?.symbol}`}
-                    height="46px"
+                    height="38px"
                   >
                     {t('addLiquidity.approve') + currencies[Field.CURRENCY_B]?.symbol}
                   </Button>
                 )}
               </RowBetween>
             )}
           <Button
-            height="46px"
+            height="38px"
```

Comment:
i think we dont need to do button height changes unless really requireed

Reply:
When I decrease the size of the drawer I had to decrease the size of the button too